src/utils/convertArrayToObject.js

```diff
@@ -0,0 +1,10 @@
+// make every item in array a key in a new object, with empty strings for values
+const convertArrayToObject = array =>
```

Comment:
Could this be given a more meaningful name?

Reply:
As a util this is exactly what it will be doing! we may use it at various points to convert different arrays!